Trees/FenwickTreeImplementation.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: Mohit Kumar
+ * Fedwick Tree Implementation in JavaScript
+ *  Fedwick Tree Implementation for finding prefix sum.
+*/
+
+function constructFenwickTree(array, n) {
+  for (let i = 1; i <= n; i++) {
+    FeneickTree[i] = 0;
+  }
+  for (let i = 0; i < n; i++) {
+    updateFenwickTree(n, i, array[i]);
+  }
+}
+
+function updateFenwickTree(n, index, value) {
+  index = index + 1;
+  while (index <= n) {
+    FeneickTree[index] += value;
+    index += index & (-index);
+  }
+}
+ 
+function getPrefixSum(index) {
+  let currSum = 0;
+  index = index + 1;
+  while (index > 0) {
+    currSum += FeneickTree[index];
+    index -= index & (-index);
+  }
+    
+  return currSum;
+}
+
+let MAX_VALUE = 1000;
```

Comment:
No global variables are allowed.

Reply:
Removed all the global variables from the file. Enclosed everything in class FenwicTree